CHANGES.rst

```diff
@@ -4,6 +4,8 @@ Changes
 1.0 (unreleased)
 ----------------
 
+* Add support for Python 3.9, 3.10, 3.11.
+
 * Drop support for Python 2.7, 3.5, 3.6.
```

Comment:
These lines should be below "Breaking changes." Or remove those sub headers if you want.

Reply:
Fixed in https://github.com/zopefoundation/z3c.autoinclude/pull/16/commits/e4ef126645177f361eb976934eef9c3b6bd7ae26